src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -45,315 +44,1079 @@ public Client(Credentials credentials) {
+     * @param options Additional options to pass along as query parameters
+     * @return Predictions response from REST API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject listPredictions(ListPredictionsOptions options)
+    throws IOException, APIException, MissingAccessTokenException {
+        HttpUriRequest request = this.createAuthorizedRequest("GET", "/predictions", options.toList());
+        String response = this.executeRequest(request);
+        return new JSONObject(response);
+    }
+
+    /**
+     *  List predictions available, calls the GET /predictions endpoint.
+     *
+     * @return Predictions response from REST API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject listPredictions() throws IOException, APIException, MissingAccessTokenException {
+        return this.listPredictions(new ListPredictionsOptions());
+    }
+
+    /**
+     * Creates a secret, calls the POST /secrets endpoint.
+
+     * @see CreateSecretOptions
+     * @param data Key-Value pairs to store secretly
+     * @param options Additional options to include in request body
+     * @return Secret response from API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject createSecret(
+        JSONObject data,
+        CreateSecretOptions options
+    ) throws IOException, APIException, MissingAccessTokenException {
+        JSONObject body = new JSONObject(){{ put("data", data); }};
+        if (options != null) {
+            body = options.addOptions(body);
+        }
+        HttpUriRequest request = this.createAuthorizedRequest("POST", "/secrets", body);
+        String jsonResponse = this.executeRequest(request);
+        return new JSONObject(jsonResponse);
+    }
+
+    /**
+     * Creates a secret, calls the POST /secrets endpoint.
+
+     * @see CreateSecretOptions
+     * @param data Key-Value pairs to store secretly
+     * @param options Additional options to include in request body
+     * @return Secret response from API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject createSecret(Map<String, String> data, CreateSecretOptions options)
+    throws IOException, APIException, MissingAccessTokenException {
+        return this.createSecret(new JSONObject(data), null);
```

Comment:
Should send options instead of null here

Reply:
Why?